lib/Minz/Extension.php

```diff
@@ -236,6 +237,21 @@ public function getUserConfiguration() {
 		return FreshRSS_Context::$user_conf->{$this->config_key}[$this->getName()];
 	}
 
+	/**
+	 * @param mixed $default
+	 * @return mixed
+	 */
+	public function getUserConfigurationValue(string $key, $default = null) {
+		if (!is_array($this->user_configuration)) {
+			$this->user_configuration = $this->getUserConfiguration();
+		}
+
+		if (array_key_exists($key, $this->user_configuration)) {
+			return $this->user_configuration[$key];
```

Comment:
What do you want to return if `$this->user_configuration[$key] === null` ?
`null` or `$default` ?

Reply:
`null` since this is the stored value.

Reply:
Ok then